scalableminds/webknossos
# Added more drawings and rebranding to UI

This PR adds more icons & drawings and rebranding to various pages within the UI

Changes:
- Applied `404` page rebranding 
- Applied `Paid Feature not available` page rebranding
- Applied `login` page rebranding with little welcome guy
- Applied `registration` page rebranding with little welcome guy
- Applied `registration page` page rebranding also the `RegistrationPageDemo` page which was forgotten the last time around
- Renamed `RegistrationFormDemo` to  `RegistrationFormWKOrg` to avoid confusion
- Removed all references to word `spotlight`. Renamed CSS classes, files and JS Components accordingly

### URL of deployed dev instance (used for testing):
- 404: https://drawings.webknossos.xyz/lol
- Sign up: https://drawings.webknossos.xyz/auth/signup
- Log in: https://drawings.webknossos.xyz/auth/login

- Make sure to downgrade the organization to "basic" plan first:
  - Feature not available: https://drawings.webknossos.xyz/tasks

### Steps to test:
- Click through all the mentioned pages above and check if styling looks correct
- Try screen resizing? Try Light/Dark Theme

### Issues:
- Contributes to https://github.com/scalableminds/webknossos/pull/6739

------
(Please delete unneeded items, merge only when none are left open)
